apps/artwork/components/meta_data/templates/price.jade

```diff
@@ -1,9 +1,6 @@
 if artwork.sale_message && !artwork.auction
   .artwork-meta-data-price
-    if artwork.price
-      .exact-price-label Price
-      .exact-listed-price= artwork.price
-      .shipping-info Shipping, tax, and services quoted by seller
-    else
-      span.price-label Price:
-      span.listed-price= artwork.sale_message
+    if artwork.sale_message
+      .sale_message= artwork.sale_message
+    if artwork.price && !artwork.availability == 'sold'
```

Comment:
I think this not(!) conditional is weird, maybe just `artwork.availability !== 'sold'`?